### PR TITLE
Optimise/add `search`[`-*`] icons

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1061.

### CODE REVIEW

**Comment by jguddas** [comment]
`search-check` is wrong, otherwise 👍

**Comment by danielbayley** [comment]
> `search-check` is wrong

Oops… fixed.

**Review by jguddas — APPROVED** [review]

**Comment by jguddas** [comment]
the x and slash are not perfectly centered.

**Comment by danielbayley** [comment]
> `search-check` is wrong

Oops… going too fast! 😅

> the x and slash are not perfectly centered.

Fixed.

File: icons/search-x.svg
Comment (karsa-mistmere) [inline icons/search-x.svg:1]: How about going 5x5 for the `x`? Still works pretty well at 100%, and I think it looks much more balanced:
image

**Review by jguddas — APPROVED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Pull request merged**